cogeo_mosaic/scripts/cli.py

```diff
@@ -92,6 +92,19 @@ def create(
         click.echo(json.dumps(mosaicjson.dict(exclude_none=True)))
 
 
+@cogeo_cli.command(short_help="Upload mosaic definition to backend")
+@click.argument("input", type=click.File(mode="r"), default="-")
+@click.option("--url", type=str, help="URL to which the mosaic should be uploaded.")
+def upload(
+    file, url,
+):
```

Comment:
does isort or black format this line like this ?

Reply:
I ran black, but maybe it didn't propagate?

Reply:
Actually it's because of the `,` after `url`